web-app-pix2info-python/src/backend/options.py

```diff
@@ -0,0 +1,78 @@
+"""
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from enum import Enum, auto
+from json import loads
+from typing import NamedTuple
+
+
+class ImageFormat(Enum):
+    """Possible formats for images sent to the frontend.
+
+    Each chosen format supports both single-frame images and animations.
+    """
+
+    AUTO = auto()
+    WEBP = auto()
+    PNG = auto()
+    GIF = auto()
```

Comment:
Does this specifically handle images that the mimetype can't be determined automatically?

Reply:
This is just a mapping of the options found in the frontend. The user can choose to download a WebP, PNG, or GIF output image (animated or not). The blot post animations were generated with the Animated/Auto options.